docs/build-from-scratch.md

```diff
@@ -21,6 +21,10 @@ The files are installed in `$GOPATH/src/github.com/heptio/ark`.
 
 ## 2. Build
 
+### Using a locally built image
+
+To ensure that your Ark deployment uses your local image rather than pulling from an image registry, you'll need to explicitly set the `spec.template.spec.containers[*].imagePullPolicy` in your deployment file to `IfNotPresent`. This file is either `examples/azure/00-ark-deployment.yaml` if you're using Azure, or `examples/common/10-deployment.yaml` for all other cloud providers.
```

Comment:
Let's make this easier. Just have them set `$REGISTRY` to an appropriate value, `$VERSION` to something other than `latest`, and tell them to use `kubectl set image`. This way they can push to a registry to support a non-local cluster (if needed), and it works for local dev too. WDYT?